packages/design-system/src/components/web-components/ds-modal-dialog/ds-modal-dialog.test.interaction.ts

```diff
@@ -13,10 +13,10 @@ Object.keys(themes).forEach((theme) => {
 
   test(`Dialog open: ${theme}`, async ({ page }) => {
     await page.goto(`${storyUrl}&globals=theme:${theme}`);
-    const elem = page.getByRole('button');
+    const elem = page.getByRole('button', { name: /Open Modal/ });
 
     await elem.click();
-    await sleep(100);
+    await sleep(200);
```

Comment:
Was the test flakey at 100ms?

Reply:
It's possible that we don't need this change anymore and that the polling fixed it. I'll run it a few more times and see if it's still flaky. I originally started changing this value when I was trying to figure out whether there was even a race condition and what was causing it.

Reply:
I've re-run just those ones quite a few times now at the original 100ms, and they pass every time. Guess we don't need this change 👍